### ISSUE TITLE
 Error: StaticInjectorError(AppModule)[ModalGalleryComponent -> KeyboardService]

### ISSUE THREAD

**Issue by Bob19891212** [Ks89/angular-modal-gallery#174]
Hi:
  I'm face this problem,when i use "ks-carousel"  in my custom module,  i will face this problem 
  "CarouselComponent.html:135 Error: StaticInjectorError(AppModule)[ModalGalleryComponent -> KeyboardService]"
  when i use in root module(appmodule),it won't appear this issue

Angular CLI: 7.0.6
Node: 8.9.4
Angular: 
... 

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.10.6
@angular-devkit/core         7.0.6
@angular-devkit/schematics   7.0.6
@schematics/angular          7.0.6
@schematics/update           0.10.6
rxjs                         6.3.3
typescript                   3.1.6

**Comment by Ks89** [comment]
Also, did you check the official live example? https://stackblitz.com/edit/angular-modal-gallery-v7

**Comment by Bob19891212** [comment]
Yes, i add the module without forRoot() in child moule,
i do a test in stackblitz, https://stackblitz.com/edit/angular-axdjwp

**Comment by Ks89** [comment]
What? forRoot in child module is wrong. This is a common pattern in Angular. Forroot() is only for the root module.
In your example I don't see any forRoot(). You aren't importing the module in your root module. 

Please check my example carefully, because it's working and you can copy the configuration. It's also explained in the official documentation website.

**Comment by Bob19891212** [comment]
you are right ,if i add the module with forRoot()  in my child module,it works

Thanks
